app/lib/three_scale/api/collection.rb

```diff
@@ -36,7 +38,21 @@ def respond_to_missing?(method_name, include_private = false)
 
   private
 
-  def pagination
-    pagination_attrs(@collection)
+  attr_reader :collection
```

Comment:
Shouldn't `representer` and `options` also be declared as `attr_reader`?
```suggestion
  attr_reader :collection, :options, :represented
```

Reply:
@josemigallas options ok, but representer no because it is not always needed so no need to put it in an instance variable at initialization